youtube-sync

```diff
@@ -85,7 +87,11 @@ function set_default_option {
 }
 
 function set_default_options {
-	set_default_option "$1" "format" "bestvideo+bestaudio/best"
+	defaultformat="$(cat "default-format")"
```

Comment:
I'd suggest the path `META/default-format` instead of `./default-format`

Reply:
besides that I'd prefer to stay consistent with how the script handles options. that would mean to *create* the option file, e.g. `set_default_global_option "default-format" "bestvideo+bestaudio/best"`.

Reply:
> to _create_ the option file

oh, I now realize you've *committed* the option file to the repo. I wouldn't do that. Besides, `cat "default-format"` refers to `./default-format`, and the yt-sync script doesn't neccessarily need to resist in the PWD.

Reply:
> I'd suggest the path `META/default-format` instead of `./default-format`
If i understand correctly the META directory is for saving the metadata about the channels and videos...i dont understand exactly why saving the config-file in an extra META directory would have any advantage. But i also don't really care about how the maintainer wants to structure the codebase. So, just let me know how and why you want it ;)